src/App.js

```diff
@@ -1,25 +1,58 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
 
 function App() {
+  
+  const [list, setList] = useState(foods);
+
+  const addFood = (food) => {
+    const foodListCopy = [...list, food]
+    setList(foodListCopy)
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+   {/* {list.map((el) => { */}
+  {/* //       return (
+  //         <div key={el.name} >
+  //           <p>{el.name}</p>
+  //           <img src={el.image} alt="foodImage" className="src" width={100} />
+  //         </div>
+  //       );
+  //     })} */}
+
+  <AddFoodForm addFood={addFood} />
+    {food.map((food) => {
```

Comment:
Here, you need to map over the State variable that stores your data.
So, in this case, you needed to map over the `list` variable that you defined earlier.